setup.py

```diff
@@ -565,6 +565,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     "apache.hive": hive,
     "apache.kylin": kylin,
     "apache.pinot": pinot,
+    "apache.presto": presto,
+    "apache.spark": spark,
```

Comment:
My intention was not to touch the setup.py file. But these changes seemed logical and safe.  Hope I am right.